frontend/lib/api/polling_station/PollingStationListProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import { PollingStation, usePollingStationListRequest } from "@kiesraad/api";
+
+export interface iPollingStationListProviderContext {
+  pollingStations: PollingStation[];
+}
+
+export const PollingStationListProviderContext = React.createContext<iPollingStationListProviderContext | undefined>(
+  undefined,
+);
+
+export interface PollingStationListProviderProps {
+  children: React.ReactNode;
+  electionId: number;
+}
+
+export function PollingStationListProvider({ children, electionId }: PollingStationListProviderProps) {
+  const { data, loading } = usePollingStationListRequest({ election_id: electionId });
+
+  if (loading) {
+    return null;
+  }
+
+  if (!data || !data.polling_stations.length) {
+    return <div id="no-polling-station-data">Er zijn nog geen stembureaus aangemaakt voor deze verkiezing.</div>;
+  }
```

Comment:
I think that when there are no polling stations created yet for the election, there should not be this sentence but just a text in place of the table and the create button should be shown (added in the next issue). Currently there will be an empty age with this sentence shown.

Reply:
I've added the empty state, without any buttons for now. The Handmatig invullen will be added for issue https://github.com/kiesraad/abacus/issues/431